src/constants/routes.ts

```diff
@@ -0,0 +1,3 @@
+export const MAIN_PAGE = "/";
+export const EVENTS_PAGE = "/events";
+export const PAGE_404 = "*";

```

Comment:
there were all ok =)

Please create `constants/navigation.ts` instead and add there
```
export const PATH = {
  MAIN_PAGE: "/",
  EVENTS_PAGE: "/events",
  PAGE_404: "*",
}

export const router = ....
``